src/vfxt/src/createUiDefinition.json

```diff
@@ -9,18 +9,18 @@
           "type": "Microsoft.Common.TextBlock",
           "visible": true,
           "options": {
-            "text": "Before creating a VM, you must create a new subscription for the Avere vFXT project, configure subscription ownership, create AAD RBAC roles for creation and cluster operation, check quotas and request an increase if needed, and accept terms for using the Avere vFXT software.",
+            "text": "Before creating a VM, you must create a new subscription for the Avere vFXT project, configure subscription ownership, create access control roles for creation and cluster operation, check quotas and request an increase if needed, and accept terms for using the Avere vFXT software.",
             "link": {
-              "label": "Read Prepare to create the Avere vFXT for detailed instructions",
-              "uri": "https://docs.microsoft.com/en-us/azure/avere-vfxt/avere-vfxt-prereqs"
+              "label": "Read 'Avere vFXT for Azure - deployment overview' for detailed instructions",
+              "uri": "https://docs.microsoft.com/en-us/azure/avere-vfxt/avere-vfxt-deploy-overview"
```

Comment:
There are implications in changing the slug (as I know you're aware). Please be sure that this change doesn't break any links to this page.